geoclue/geoclue/gc-provider.c

```diff
@@ -189,7 +189,25 @@ gc_provider_remove_client (GcProvider *provider, const char *client)
 	(*pcount)--;
 	if (*pcount == 0) {
 		g_hash_table_remove (priv->connections, client);
+        }
```

Comment:
Looks like here and later on g_hash_table_remove there's some hard tab vs space issue on the indentation level.

Reply:
I hope Emacs caught all tab vs. spaces conflicts.